src/main/scala/chiseltest/backends/VPIInterface.scala

```diff
@@ -0,0 +1,427 @@
+package chiseltest.backends
```

Comment:
This looks similar to the old SimApiInterface.scala. Was this code significantly refactored and should be reviewed as new code, or should it have `legacy` as part of the package name to indicate old code?

Reply:
It's was legacy code.
I'll make it legacy and refactor in the future PR.